packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/summary/translations/Messages_fr_FR.json

```diff
@@ -9,6 +9,7 @@
   "dedicatedCloud_datacenter_drp_confirm_create_success_part_two": "Vous avez reçu votre mot de passe ZSSP sur l’adresse email {{email}}.",
   "dedicatedCloud_datacenter_drp_confirm_creation_pending": "Votre PRA Zerto est en cours d’activation. L’opération peut prendre jusqu’à une heure de délai.",
   "dedicatedCloud_datacenter_drp_confirm_creation_pending_task": "N’hésitez pas à vérifier la tâche enableDisasterRecovery du PCC principal pour suivre son avancement.",
+  "dedicatedCloud_datacenter_drp_confirm_order": "Votre <a target=\"_blank\" href=\"{{billUrl}}\" rel=\"noopener\">bon de commande</a> a été généré avec succès. Une fois le paiement de celui-ci effectué, vous pourrez vérifier la tâche enableDisasterRecovery dans l‘onglet Opérations, pour suivre l‘activation de l‘option.",
```

Comment:
is the task enableDisasterRecovery not translated on our side?

Reply:
No it is not translated on our side (ex: https://github.com/ovh/manager/blob/master/packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/drp/summary/translations/Messages_it_IT.json#L11)

Reply:
can't we translate it ? ^^

Reply:
Good question :). I think we should think about it for every task, since all the task are displayed as they are returns by the API. (Name, and comment for each of them)